README.md

```diff
@@ -33,29 +32,39 @@
       images: '[
                 {
                   "name": "sample-golang",
-                  "repository": "registry.digitalocean.com/sample-go/add_sample",
```

Comment:
Should we leave this format in the README under a `deprecated` section?

Reply:
I'd prefer not to, it can be confusing and I do not want it to proliferate further. I did make the GH Action output a warning.